packages/e2e-tests/vite-ssr-esm/package.json

```diff
@@ -5,9 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "node server",
-    "build": "run-s build:client build:server",
-    "build:client": "vite build --ssrManifest .vite/ssr-manifest.json --outDir dist/client",
-    "build:server": "vite build --ssr src/entry-server.js --outDir dist/server",
+    "build": "vite build --app",
```

Comment:
for `--watch` to work swiched this over to environment api builder, the previous scripts would not work as they relied on being called in sequence.